website/_sass/custom/custom.scss

```diff
@@ -17,6 +17,7 @@
     vertical-align: text-bottom;
 }
 
+// Center and shrink the footer logo.
```

Comment:
@Stephan202 isn't this overkill? You're describing the code into a comment here.
- `footer` tells us already where it is
- `#logo` tells us this is about the logo
- `width` already tells us that we're resizing
- `margin: 0 auto` and `display: block` tell us we're centering the image

Reply:
To be frank, I was wondering the same when I wrote it. I considered describing the delta (what would it look like otherwise?) but that can get out of sync. I think this is a combo if me being unfamiliar with CSS (it's been years since I seriously used it), and wanting to describe why we override a theme default. I'm okay with dropping it.